changelogs/unreleased/4109-erwbgy-small.md

```diff
@@ -0,0 +1 @@
+Allow the contour --ingress-class-name value to be a comma-separated list of classes to match against.  Contour will process Ingress and HTTPProxy objects with any of the specified ingress classes.
```

Comment:
Even though the ContourConfiguration CRD is alpha, I think it's worth calling out in the release note that the name (if you accept my previous comment) and type of the ingress class field are changing in a non-backwards-compatible way.

Reply:
Updated release note.